translations/German/special-terms.md

```diff
@@ -0,0 +1,275 @@
+term. For this reason, use "Integrationen" instead of "Integrations"
+when speaking of multiple integrations in German. There aren't many
+German sources available for this translation, but "Integration" has
+the same meaning in German and English.
+
+*"Integration/-en" (Transifex)*
+
+* Notification - **Benachrichtigung**
+
+Nice and easy. Other translations for "notification" like
+"Erwähnung", "Bescheid" or "Notiz" don't fit here.
+
+*"Benachrichtigung" (Facebook, Gmail, Transifex, Wikipedia)*
+
+* Alert Word - **Signalwort**
+
+This one is tricky, since one might initially think of "Alarmwort" as a
+proper translation. "Alarm", however, has a negative connotation, people
+link it to unpleasant events. "Signal", on the other hand, is neutral,
+just like "alert word". Nevertheless, [Linguee](
+http://www.linguee.de/deutsch-englisch/search?source=auto&query=alert+word)
+shows that some websites misuse "Alarm" for the translation.
+
+*"Signalwort" (Transifex), "Wort-Alarm" (Linguee)*
+
+* View - **View** (Developer documentation)
+
+Since this is a Zulip-specific term for 
+> every path that the Zulip server supports (doesn’t show a 404 page for),
+
+and there is no German equivalent, talking of "Views" is preferable in the
+developer documentation and makes it easier to rely on parts of the German
+*and* parts of the English documentation.
+
+* View - **Ansicht** (User documentation)
+
+For the user documentation, we want to use "Ansicht" instead of
+"view", as "Ansicht" provides a translated description for what you
+think of when hearing "view". "Ansicht" is not desirable for the
+developer documentation, since it does not emphasize the developing
+aspects of views (in contrast to anglicisms, which Germans often link
+to IT-related definitions).
+
+*"Ansicht" (Transifex)*
+
+* Home - **Startseite**
+
+Nice and easy. "Zuhause" obviously doesn't fit here ;).
+
+*"Startseite" (Facebook, Transifex)*
+
+* Emoji - **Emoji**
+
+"Emoji" is the standard term for Emojis. Any other
+Germanized translation like "Bildschriftzeichen" (which exists!)
+would sound stiff and outdated. "Emoticon" works as well,
+but is not that common in German.
+
+*"Emoji" (Facebook, WhatsApp), "Emoticon" (Google+)*
+
+## Phrases (Ausdrücke)
+
+* Subscribe/Unsubscribe - **Abonnieren/Deabonnieren**
+
+This translation is unambiguous.
+
+*"Deabonnieren" (Youtube, Transifex)*
+
+* Narrow to - **Begrenzen auf**
+
+Transifex has two different translations for "Narrow to" -
+"Schränke auf ... ein." and "Begrenze auf ... ." Both sound
+a bit strange to a German speaker, since he would expect
+grammatically correct sentences when using the imperative
+(e.g. "Schränke diesen Stream ein auf ... .") Since this would
+be too long for many labels, the infinitive "begrenzen auf"
+is preferable. "einschränken auf" sounds equally good, but
+Transifex shows more use cases for "begrenzen auf".
+
+*"Schränke auf ... ein." (Transifex) "Begrenze auf ... ." (Transifex)*
+
+* Filter - **Filtern**
```

Comment:
Some words in English can be both nouns and verbs - we should remember that when creating the main PR for German style guides.